google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/RemoveEntireSitelinkCampaignExtensionSetting.java

```diff
@@ -0,0 +1,192 @@
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+      params.campaignId = Long.parseLong("INSERT_CAMPAIGN_ID_HERE");
+    }
+
+    GoogleAdsClient googleAdsClient = null;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      System.exit(1);
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      System.exit(1);
+    }
+
+    try {
+      new RemoveEntireSitelinkCampaignExtensionSetting()
+          .runExample(googleAdsClient, params.customerId, params.campaignId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+      System.exit(1);
+    }
+  }
+
+  /** Runs the example. */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long campaignId) {
+    // Creates an operation to remove a campaign extension setting.
+    CampaignExtensionSettingOperation extensionSettingOperation =
+        CampaignExtensionSettingOperation.newBuilder()
+            .setRemove(
+                ResourceNames.campaignExtensionSetting(
+                    customerId, campaignId, ExtensionType.SITELINK))
+            .build();
+
+    // Retrieves all sitelink extension feed items for a customer and campaign.
+    List<String> extensionFeedItems =
+        getSitelinkExtensionFeedItems(googleAdsClient, customerId, campaignId);
+
+    // Constructs operations to remove the extension feed item operations.
+    List<MutateOperation> extensionFeedItemOperations =
+        extensionFeedItems.stream()
+            .map(
+                sitelinkExtension ->
+                    CampaignExtensionSettingOperation.newBuilder()
+                        .setRemove(sitelinkExtension)
+                        .build())
+            .map(
+                extensionOperation ->
+                    MutateOperation.newBuilder()
+                        .setCampaignExtensionSettingOperation(extensionOperation)
```

Comment:
This is probably wrong.
The mutate operations we're building should should be composed of 1) campaign_extension_setting_operation and 2) extension_feed_item_operation.

It looks like you're building two mutate operations that are both "campaign_extension_setting_operation"